docs/ai/how-to/swap-between-openai-and-aoai.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Swap between OpenAI and AOAI"
+description: "Demonstrates how to swap between OpenAI and Azure OpenAI (AOAI) using the Semantic Kernel SDK for .NET."
+author: haywoodsloan
+ms.topic: how-to
+ms.date: 04/09/2024
+
+#customer intent: As a .NET developer, I want to swap between OpenAI and Azure OpenAI (AOAI) so that I can choose the AI service that's best for me.
+
+---
+
+# Swap between OpenAI and AOAI
+
+This article demonstrates how to swap between the OpenAI and [Azure OpenAI](/azure/ai-services/openai/overview) (AOAI) services using the [Semantic Kernel SDK](/semantic-kernel/overview). Additionally, it shows how to switch your .NET projects to the Semantic Kernel SDK from the [AOAI client library](/dotnet/api/overview/azure/ai.openai-readme).
+
+## Prerequisites
+
+* An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* [.NET SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
+* [`Microsoft.SemanticKernel` NuGet package](https://www.nuget.org/packages/Microsoft.SemanticKernel)
+* [Create and deploy an Azure OpenAI Service resource](/azure/ai-services/openai/how-to/create-resource)
+
```

Comment:
Do we want to include a little bit more intro context here that describes some reasons why they might want to switch, or some conceptual differences/steps between the two to consider for any migration?

Reply:
Since this is a how-to I first focused on the scenarios that Bruno specified. It might make sense to describe some benefits in switching to SK, which I can grab from their docs. 

As for the conceptual differences between the two, I think that would need a much deeper dive to understand the two SDKs better first.

Reply:
These are good points, yes maybe just a little bit info of the benefits or reasons you might want to switch would be good since it is a focused how-to.